source/PlayerInfo.cpp

```diff
@@ -545,11 +545,15 @@ void PlayerInfo::IncrementDate()
 		}
 	}
 
-	// Check if any missions have failed because of deadlines.
+	// Check if any missions have failed because of deadlines and do on daily actions if not.
 	for(Mission &mission : missions)
+	{
 		if(mission.CheckDeadline(date) && mission.IsVisible())
-			Messages::Add("You failed to meet the deadline for the mission \"" + mission.Name() + "\"."
-				, Messages::Importance::Highest);
+			Messages::Add("You failed to meet the deadline for the mission \"" + mission.Name() + "\".", 
```

Comment:
Trailing whitespace.
```suggestion
			Messages::Add("You failed to meet the deadline for the mission \"" + mission.Name() + "\".",
```